pkg/apicontracts/apiresourcecontracts/resourcedef_vulnerabilitywhitelist.go

```diff
@@ -0,0 +1,15 @@
+package apiresourcecontracts
+
+import "github.com/NorskHelsenett/ror/pkg/rorresources/rortypes"
+
+type ResourceVulnerabilityWhitelist struct {
+	ApiVersion string                             `json:"apiVersion"`
+	Kind       string                             `json:"kind"`
+	Metadata   ResourceMetadata                   `json:"metadata"`
+	Spec       ResourceVulnerabilityWhitelistSpec `json:"spec"`
+}
+
+type ResourceVulnerabilityWhitelistSpec struct {
+	Owner   rortypes.RorResourceOwnerReference `json:"owner"`
+	Message string                             `json:"message"`
```

Comment:
Missing CVE property, or perhaps a list of cve's? And potensially a reference to a risk and vulnerability report

Reply:
Vulnerability whitelist is no longer a resource, treatment happens in Cluster Vulnerability Report resource